### ISSUE TITLE
Visualization does not render User Input

### ISSUE THREAD

**Issue by loomsen** [RasaHQ/rasa#7600]
**Rasa version**:
2.1.2

**Python version**:
Python 3.7.9

**Operating system** (windows, osx, ...):
rasa Docker image

**Issue**:
When using `rasa visualize` the rendered graphs don't have any user input from the stories.

![Selection_639](https://user-images.githubusercontent.com/302524/102635194-48899800-4153-11eb-9264-4b06ee02aab0.png)

Obviously, without the user input, the graph is pretty useless. This used to work fine for us up until rasa 2.0.

**Error (including full traceback)**:
There is no error, command runs fine
```
# rasa visualize -s data/core/common.story.yml 
┌────────────────────────────────────────────────────────────────────────────────┐
│ Rasa Open Source reports anonymous usage telemetry to help improve the product │
│ for all its users.                                                             │
│                                                                                │
│ If you'd like to opt-out, you can use `rasa telemetry disable`.                │
│ To learn more, check out https://rasa.com/docs/rasa/telemetry/telemetry.       │
└────────────────────────────────────────────────────────────────────────────────┘
2020-12-18 16:02:35 INFO     rasa.core.visualize  - Starting to visualize stories...
Processed story blocks: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00, 2593.88it/s, # trackers=1]
2020-12-18 16:02:36 INFO     rasa.core.visualize  - Finished graph creation. Saved into file:///app/graph.html
```

**Command or request that led to error**:
```
rasa visualize
```

**Content of configuration file (config.yml)** (if relevant):
```yml
language: de
pipeline:
- name: SpacyNLP
- name: SpacyTokenizer
- name: SpacyFeaturizer
- name: RegexFeaturizer
- name: LexicalSyntacticFeaturizer
- name: CountVectorsFeaturizer
- name: CountVectorsFeaturizer
  analyzer: char_wb
  min_ngram: 1
  max_ngram: 4
- name: DIETClassifier
  epochs: 100
  hidden_layers_sizes:
    text: [256, 128]
  number_of_transformer_layers: 0
  weight_sparsity: 0
  intent_classification: True
  entity_recognition: False
  use_masked_language_model: False
  BILOU_flag: False
- name: EntitySynonymMapper
- name: FallbackClassifier
  threshold: 0.3
  ambiguity_threshold: 0.1
policies:
- name: AugmentedMemoizationPolicy
- name: TEDPolicy
  epochs: 10
- name: RulePolicy
  core_fallback_threshold: 0.5
  core_fallback_action_name: action_fallback
```

**Content of story file**:
```yml
version: "2.0"
stories:
- story: intro
  steps:
  - intent: common.greetings
  - action: utter_greet
- story: ciao
  steps:
  - intent: common.goodbyes
  - action: utter_goodbye
- story: thankyou
  steps:
  - intent: common.thankyous
  - action: utter_youre_welcome
- story: User logged in
  steps:
  - intent: common.handover
  - slot_was_set:
    - channel: whatsapp
  - slot_was_set:
    - logged_in: true
  - action: utter_trigger_handover
 ```

**Comment by sara-tagger** [comment]
Thanks for the issue, @amn41 will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by amn41** [comment]
thanks for filing this bug. Will add it to the contributor board in case someone wants to pick it up, I think it might be a good one for a new contributor.

**Comment by SirBenedick** [comment]
I am experiencing the same bug with rasa 2.5.0 (and previously with 2.2.10).
It seems to be related to the way I store my stories and rules. ```rasa visualize --debug``` does not recognize the yaml file as a valid _rasa_yml_ file, even though I am experiencing no other problem with the file format.

snapshot of the ```rasa visualize --debug``` output:
```
❯ rasa visualize --debug
2021-04-27 18:28:28 DEBUG    rasa.shared.nlu.training_data.loading  - Training data format of 'data/nlu/general.yml' is 'rasa_yml'.
2021-04-27 18:28:28 DEBUG    rasa.shared.nlu.training_data.loading  - Training data format of 'data/nlu/inform.yml' is 'rasa_yml'.
2021-04-27 18:28:28 DEBUG    rasa.shared.nlu.training_data.loading  - Training data format of 'data/nlu/use_cases.yml' is 'rasa_yml'.
2021-04-27 18:28:28 DEBUG    rasa.shared.nlu.training_data.loading  - Training data format of 'data/stories/000_general.yml' is 'unk'.
2021-04-27 18:28:28 DEBUG    rasa.shared.nlu.training_data.loading  - Training data format of 'data/stories/001_introduction.yml' is 'unk'.
...
```

content of the ```data/stories/001_introduction.yml``` file:
```
version: "2.0"

stories:
  - story: 001 - Greet user...
    steps:
      - intent: greet
      - action: action_001_greet_user
      - action: utter_how_can_i_help
```

**Comment by benjenis** [comment]
has this bug been resolved?

**Comment by loomsen** [comment]
Nope, not as far as I can tell and see.

**Comment by bauyrjanj** [comment]
@amn41 will this bug be addressed?

**Comment by arunbg** [comment]
Still an issue in Rasa 2.8.2

**Comment by hsm207** [comment]
Closing this since it has been fixed in 3.0
